reproducibility_project/waterspce_nist_subproject/init.py

```diff
@@ -0,0 +1,169 @@
+"""Initialize signac statepoints."""
+import itertools
+import os
+
+import numpy as np
+import signac
+import unyt as u
+from numpy import ModuleDeprecationWarning
+
+
+def dict_product(dd):
+    """Return the product of the key/values of a dictionary."""
+    keys = dd.keys()
+    for element in itertools.product(*dd.values()):
+        yield dict(zip(keys, element))
+
+
+molecules = [
+    "waterSPCE",
+]
+replicas = range(16)
+simulation_engines = [
+    "gromacs",
+    "lammps-VU",
+]
+md_engines = ["gromacs", "hoomd", "lammps-VU", "lammps-UD"]
+mc_engines = ["cassandra", "mcccs", "gomc"]
+forcefields = {}
+r_cuts = {}
+cutoff_styles = ["hard"]
+long_range_correction = ["energy_pressure"]
+for key in molecules:
+    if "UA" in key:
+        if "benz" not in key:
+            forcefields[key] = "trappe-ua"
+        else:
+            forcefields[key] = "benzene-ua"
+        r_cuts[key] = 14.0 * u.angstrom
+    elif "SPCE" in key:
+        forcefields[key] = "spce_nist"
+        r_cuts[key] = 9 * u.angstrom
+    else:
+        forcefields[key] = "oplsaa"
+        r_cuts[key] = 10 * u.angstrom
+g_per_cm3 = u.g / (u.cm * u.cm * u.cm)
+masses = {
+    "waterSPCE": [18.015324] * u.amu,
+}
+init_density_liq = {
+    "waterSPCE": [0.998] * g_per_cm3,
+}
+init_density_vap = {
+    "waterSPCE": [None],
+}
+temperatures = {
```

Comment:
Do we want to run this many simulations? We can probably get away with just running at 300K I think, especially if the results look good.

Reply:
I was thinking running the whole set can help you see if the small difference could create different effect at different temperature, but I am down to running just one

Reply:
For LAMMPS, presumably the slower engine, the production run time was Total wall time: 10:41:30 for 5 ns. It took about 7 ns to equilibrate so that would put us at about 24 hours per simulation, so we could presumably finish these in about 2-3 days for the 16 replicates, assuming low queue times.

Reply:
the queue is pretty empty (for now). I have been running the gromacs stuff, though with the slightly diff params again. I can stop those and rerun when we agree upon a final number